src/status.rs

```diff
@@ -79,11 +79,11 @@ pub struct RadioShow {
 }
 
 /// The main Space API status object.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, RustcDecodable)]
 pub struct Status {
 
     // Hackerspace properties
-    pub api: &'static str,
```

Comment:
Oh this is indeed a breaking change, forget the previous comment about *not* increasing the version to `0.2.0`